test/e2e/benchmark/manifest.go

```diff
@@ -68,6 +68,14 @@ type Manifest struct {
 	// consensus parameters
 	MaxBlockBytes int64
 
+	// other configs
```

Comment:
is this line necessary considering there's another `//other configs` slightly below?

```suggestion
```